src/libraries/System.Collections.Immutable/src/System/Collections/Frozen/String/KeyAnalyzer.cs

```diff
@@ -221,7 +199,7 @@ private static bool ContainsAnyLetters(ReadOnlySpan<char> s)
 #endif
         }
 
-        private static bool HasSufficientUniquenessFactor(HashSet<string> set, ReadOnlySpan<string> uniqueStrings)
+        private static bool HasSufficientUniquenessFactor(HashSet<string> set, string[] uniqueStrings)
```

Comment:
Once again, we know it's a string[] from the source calls, and switching this to `IEnumerable<string>` would lose access to the `Length`, so don't do that.